integration-tests/byoid/integration_test.go

```diff
@@ -37,9 +37,14 @@ import (
 	"log"
 	"net/http"
 	"net/http/httptest"
+	neturl "net/url"
 	"os"
+	"regexp"
 	"testing"
 
+	"golang.org/x/oauth2/google"
+
+	"google.golang.org/api/dns/v1"
```

Comment:
This import is listed twice.

Reply:
That's bizarre, it isn't listed twice locally.  Let me try and fix that.